SPECS/Cython/Cython.spec

```diff
@@ -1,79 +1,396 @@
-%{!?python2_sitelib: %define python2_sitelib %(python2 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
-Summary:       C extensions for Python
-Name:          Cython
-Version:       0.28.5
-Release:       7%{?dist}
-Group:         Development/Libraries
-License:       ASL 2.0
-URL:           https://cython.org
-#Source0:      https://github.com/cython/cython/archive/%{version}.tar.gz
-Source0:       %{name}-%{version}.tar.gz
-Patch0:        fix_abc_tests.patch
-Vendor:        Microsoft Corporation
-Distribution:  Mariner
-BuildRequires: python2-devel
-BuildRequires: python2-libs
-BuildRequires: python-xml
-Requires:      python2
-
-%description
-Cython is an optimising static compiler for both the Python programming language and the extended Cython programming language (based on Pyrex). It makes writing C extensions for Python as easy as Python itself.
+%{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
+%define python3_sitearch %(python3 -c "from distutils.sysconfig import get_python_lib; import sys; sys.stdout.write(get_python_lib(1))")
+
+%global upname cython
+
+Name:           Cython
+Version:        0.29.13
+Release:        6%{?dist}
+Summary:        Language for writing Python extension modules
+Vendor:         Microsoft
+Distribution:   Mariner
+License:        ASL 2.0
+URL:            https://www.cython.org
+#Source0:       https://github.com/%{upname}/%{upname}/archive/%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+
+BuildRequires:  gcc
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-xml
+
+%global _description \
+Cython is an optimising static compiler for both the Python programming language and the extended Cython programming language (baded on Pyrex). It makes writing C extensions for Python as easy as Python itself.}
+
+%description %{_description}
+
+%package -n python3-%{name}
+Summary:        %{summary}
+%{?python_provide:%python_provide python3-%{name}}
+Provides:       %{name} = %{version}-%{release}
+Provides:       %{name}%{?_isa} = %{version}-%{release}
+Obsoletes:      %{name} < %{version}-%{release}
+
+%description -n python3-%{name} %{_description}
+
+Python 3 version.
 
 %prep
-%setup -q -n cython-%{version}
-%patch0 -p1
+%setup -n %{upname}-%{version}
 
 %build
-python2 setup.py build
+python3 setup.py build
 
 %install
-python2 setup.py install --skip-build --root %{buildroot}
 
-%check
```

Comment:
Why are we removing the `%check` section? We shouldn't be removing tests, unless the new version of the package doesn't have them anymore.